src/cwe_checker_lib/src/intermediate_representation/program.rs

```diff
@@ -49,3 +50,34 @@ impl Program {
         None
     }
 }
+
+impl fmt::Display for Program {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        for Term { tid, term: sub } in self.subs.values() {
+            writeln!(
+                f,
+                "SUB [{}] n:{} e:{}",
```

Comment:
Why shorthening name and entry point to `n` and `e`?  We have enough space for a few more characters. And the abbreviations might confuse people new to the project.